core/controllers/base.py

```diff
@@ -578,3 +578,17 @@ def is_csrf_token_valid(cls, user_id, token):
             return False
         except Exception:
             return False
+
+
+class CsrfTokenHandler(BaseHandler):
```

Comment:
With the introduction of this handler, shouldn't we be dropping the existing functionality for setting the csrf token in render_template()?